stable/kiam/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: kiam
-version: 1.2.0
-appVersion: 2.8
+version: 1.3.0-rc1
+appVersion: 3.0-rc1
```

Comment:
Is there a reason you're updating to a release candidate rather than a final release? We usually want final releases in here.

Reply:
@mattfarina here is the bug https://github.com/uswitch/kiam/pull/86
as you see there are several issues related to it

Reply:
I'm kind of inclined to leave the new version at `1.3.0` as there are no breaking changes in the chart values.
We do need some way of saying "1.3 only works with kiam 3.0" etc.
Is the README the correct place for that?